listeners.js

```diff
@@ -192,5 +198,43 @@ module.exports = function startListeners(ssb, onError) {
         }
       )
     )
+
+    // re-add missing people to a new epoch if the epoch creator didn't add everyone but they added us.
+    // we're only doing this for the preferred epoch atm
+    pull(
+      ssb.tribes2.list({ live: true }),
+      pull.map((group) =>
+        pull(
+          getPreferredEpoch.stream(group.id, { live: true }),
```

Comment:
This works, but I worry it means opening N listening streams for N groups, and `getPreferredEpoch` live is a lot.
I think we can replace all this with one listener for `group/init` messages aye? then pull the `groupId` out of the recps? 

could be a TODO / issue as this is working

Reply:
i think i did this partially because i wanted to check if the epoch was preferred anyway, so felt like i could do it this way to both get the epoch and check at the same time